core/src/main/java/org/kiwix/kiwixmobile/core/main/AddNoteDialog.java

```diff
@@ -98,6 +99,7 @@
   @Inject SharedPreferenceUtil sharedPreferenceUtil;
   @Inject ZimReaderContainer zimReaderContainer;
   @Inject protected AlertDialogShower alertDialogShower;
+  @Inject FileLogger fileLogger;
```

Comment:
there should be no changes to this file

Reply:
Oh my bad, i mistakenly edited this file.